api_tests/tests/ether_halight.ts

```diff
@@ -13,11 +13,9 @@ it(
         const bodies = (await SwapFactory.newSwap(alice, bob))
             .hanEthereumEtherHalightLightningBitcoin;
 
-        // Bob needs to know about the swap first because he is not buffering incoming announcements about swaps he doesn't know about
-        await bob.createSwap(bodies.bob);
-        await sleep(500);
-
         await alice.createSwap(bodies.alice);
+        await sleep(500);
+        await bob.createSwap(bodies.bob);
```

Comment:
:thinking: Instead of replacing this test, can we just add a dry test that ensures that creating a swap in the opposite order succeeds?

Reply:
This is currently not possible with the current e2e test framework.

Doing https://github.com/comit-network/comit-rs/issues/2589 to make it possible.